akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -272,7 +269,9 @@ object ActorMaterializerSettings {
       fuzzingMode = config.getBoolean("debug.fuzzing-mode"),
       autoFusing = config.getBoolean("auto-fusing"),
       maxFixedBufferSize = config.getInt("max-fixed-buffer-size"),
-      syncProcessingLimit = config.getInt("sync-processing-limit"))
+      syncProcessingLimit = config.getInt("sync-processing-limit"),
+      verboseLogErrors = config.getBoolean("verbose-log-errors ")
```

Comment:
I'm not convinced that it's the right approach to define one single property for this. We already have `LogLevels` and shouldn't this also be an attribute for more fine grained control. Perhaps it even belongs in the existing `LogLevels` attribute?

I noticed a related issue when working on the Artery TCP implementation. I wanted to suppress logging that is done in `reportStageError`. In my case it was for the MergeHub, but there can be other stages. Therefore I changed `reportStageError` to look at the `LogLevels` attribute.
https://github.com/akka/akka/pull/23828/files#diff-20ce84dc90488608e4a75c28b5ed0dbdR348